### PR TITLE
fix(lint): dumpling/export/sql_type.go

### PR DESCRIPTION
**Pull request by tangjingyu97**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: build/nogo_config.json [hunk 1]
```diff
@@ -504,7 +504,6 @@
   },
   "revive": {
     "exclude_files": {
-      "pkg/parser/parser.go": "parser/parser.go code",
```
Comment (hawkingrei) [inline build/nogo_config.json:507]: It is generated by tools and it is too big to linter.

Comment (hawkingrei) [reply build/nogo_config.json:507]: so it is skipped.

**Review by hawkingrei — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**